source/simulator/helper_functions.cc

```diff
@@ -23,7 +23,16 @@
 #include <aspect/melt.h>
 #include <aspect/global.h>
 
+#include <aspect/geometry_model/interface.h>
+#include <aspect/heating_model/interface.h>
 #include <aspect/heating_model/adiabatic_heating.h>
+#include <aspect/material_model/interface.h>
+#include <aspect/particle/generator/interface.h>
+#include <aspect/particle/integrator/interface.h>
+#include <aspect/particle/interpolator/interface.h>
+#include <aspect/particle/output/interface.h>
+#include <aspect/prescribed_stokes_solution/interface.h>
+#include <aspect/postprocess/visualization.h>
```

Comment:
I'll add that there is essentially no cost associated with including a file twice -- the compiler caches which files it (i) has included already, and (ii) that are guarded by include guards.